### PR TITLE
Add WithRankDecoration and fix the mcv deploy voice

### PR DESCRIPTION
**Pull request by abcdefg30**
The `Attack` voices just didn't make sense to me.

### CODE REVIEW

File: mods/ra2/rules/proxy-actors.yaml [hunk 1]
```diff
@@ -5,3 +5,4 @@ barracks.infiltrated:
 warfactory.infiltrated:
 	AlwaysVisible:
 	ProvidesPrerequisite:
+
```
Comment (phrohdoh) [inline mods/ra2/rules/proxy-actors.yaml:8]: Let's be careful not to go back and forth on these. How about we add them as the utility does, but never remove them?

**Comment by phrohdoh** [comment]
I'd like to sensibly squash the WIP commits before merging this.

File: mods/ra2/rules/proxy-actors.yaml (hunk 1 above)
Comment (abcdefg30) [inline mods/ra2/rules/proxy-actors.yaml:8]: This change was done by the utility, so I'd say keep it.

**Comment by phrohdoh** [comment]
@abcdefg30 please rebase this so I can test and merge.

Also, please fix `zep`'s voice as I forgot it in the fixup commit.

[Red Alert 2](https://github.com/Phrohdoh/OpenRA) is live! :tada:

**Edit**: and broken! :disappointed:

**Comment by abcdefg30** [comment]
Rebased.

**Comment by GraionDilach** [comment]
Those glitches might be fixed with a reimporting of the maps, via pchote's ts-map-importer branch.

**Comment by phrohdoh** [comment]
@GraionDilach yep, that was discussed this morning.

**Comment by phrohdoh** [comment]
This certainly changed `Attack` to `Move`, but I still get `Target sighted!`, `Attacking!` etc., please fix.

**Comment by phrohdoh** [comment]
Hmm, it seem us rebasing `ra2#master` onto `openra/bleed` is going to force PR authors to manually remove duplicate commits over and over.

We should find a way to remedy this.

**Comment by abcdefg30** [comment]
Updated.

File: mods/ra2/rules/proxy-actors.yaml (hunk 1 above)
Comment (penev92) [inline mods/ra2/rules/proxy-actors.yaml:8]: Hint: if you remove it, the utility will keep adding it every time you run it. It's a known bug/annoyance.
I'd suggest keeping it to reduce future diff noise.

Comment (phrohdoh) [inline mods/ra2/rules/proxy-actors.yaml:8]: That is why I said..

> How about we add them as the utility does, but never remove them?

**Comment by phrohdoh** [comment]
~~The rank automatically jumps to fully promoted. Is that intentional or just a side-effect of our values not being balanced properly?~~

It was just a coincidence.

File: mods/ra2/rules/proxy-actors.yaml (hunk 1 above)
Comment (penev92) [inline mods/ra2/rules/proxy-actors.yaml:8]: Eh, derp. Somehow I totally missed that :S

**Pull request merged**